packages/runner/bin/compile-config.ts

```diff
@@ -5,12 +5,13 @@
 
 import {compileConfig} from "componentsjs";
 import {createReadStream} from "fs";
-import {Stream} from "stream";
 import {sep} from "path";
+import {Stream} from "stream";
 
 const args = process.argv.slice(2);
 
 if (!args.length) {
+  // tslint:disable-next-line:no-console
```

Comment:
Idem for `bin`; let's have a less strict `.tslint` that allows `console`.
Example for instance in https://github.com/RubenVerborgh/LDflex/blob/master/test/.eslintrc